pytorch_sphinx_theme/footer.html

```diff
@@ -10,7 +10,22 @@
     </div>
   {% endif %}
 
-  <hr/>
+  {% if theme_pytorch_project == 'tutorials' %}
+
+    <hr class="helpful-hr hr-top">
+      <div class="helpful-container">
+        <div class="helpful-question">Was this helpful?</div>
+        <div class="helpful-question yes-link" data-behavior="was-this-helpful-event" data-response="yes">Yes</div>
```

Comment:
I assume `was-this-helpful-event` is defined somewhere already.

Reply:
`was-this-helpful-event` is used in this [tutorials PR](https://github.com/pytorch/tutorials/pull/443) to track the event via FB pixels and Google Analytics.

Reply:
Oh right - we just hadn't merged that yet 😄